README.md

```diff
@@ -57,7 +57,7 @@ For other distros, you'll need to build and install from source:
 1.  Install build dependencies.
 
     * RHEL/CentOS/Fedora: `gcc` `automake` `autoconf` `openssl-devel`
-    * Debian/Ubuntu: `gcc` `automake` `autoconf` `libssl-dev`
+    * Debian/Ubuntu: `gcc` `automake` `autoconf` `libssl-dev` `pkg-config`
```

Comment:
I suspect `pkg-config` should be added to build dependencies of all platforms: it has to be possible to install `automake`/`autoconf` without `pkg-config` on most if not all platforms, even though that might occur seldom.

Reply:
I have added pkg-config as dependency, but how do I mark this request in github as solved?